### ISSUE TITLE
BUG: Canary build is not installed properly on brand new machines

### ISSUE THREAD

**Issue by gayathrit12** [dequelabs/axe-core-npm#868]
### Product

cli

### Product Version

4.8.1-22b3a9d.0

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
`axe` should work

#### Actual
`axe is not recognized as an internal or external command, operable program or batch file.`

#### How to Reproduce
1. Created a directory
2. Created package.json ( npm init -y)
3. Installed axe-core/cli globally (npm install @axe-core/cli@next -g)
4. Installed axe-core/cli locally (npm install @axe-core/cli@next)
5. Run `axe --help` or `axe --version` and verify

### Issue:
Getting this error `axe is not recognized as an internal or external command, operable program or batch file.` when ran `axe --help` or `axe --version`

### Screenshot:
![image](https://github.com/dequelabs/axe-core-npm/assets/102580833/a355e912-f85a-4e95-9017-7ef8f0a7c83d)

#### Additional context
When navigated to global directory it worked fine.
`cd C:\...\AppData\Roaming\npm\`

### Screenshot:
![image](https://github.com/dequelabs/axe-core-npm/assets/102580833/6160f997-ad37-4a47-8335-fb35f85c0145)

### Note: 
Existing users not facing this issue.

## Environment:
| **Label**                         | **Value** |
| -------------------------- | ------ |
| **Product**                | _axe-core-npm/cli_|
|**Version** | _4.8.1-22b3a9d.0_ |
| **OS-Details**      | _MAC- Intel Core i7 - 11.6.8  & Windows11 _ |
|**Browser Details**|_Chrome Version 117.0.5938.150 (Official Build) (64-bit) & Firefox 117.0_|

**Comment by Zidious** [comment]
IIRC for Windows, globally installing the CLI will not set the use of the CLI globally using the keyword `axe`. It requires some manual steps by adding it to your [systems `$PATH`](https://learn.microsoft.com/en-us/previous-versions/office/developer/sharepoint-2010/ee537574(v=office.14)).

**Comment by padmavemulapati** [comment]
@Zidious , yes after setting the `PATH` to `Environment variable` , this is working as expected.

**Comment by gayathrit12** [comment]
This is working as intended.

## Screenshot:
![image](https://github.com/dequelabs/axe-core-npm/assets/102580833/51c66a48-5bd5-49c7-89c7-c572f6aa216e)


## Environment:
| **Label**                         | **Value** |
| -------------------------- | ------ |
| **Product**                | _axe-core-npm/cli_|
|**Version** | _4.8.1-5bfd440.0_ |
| **OS-Details**      | _MAC- Intel Core i7 - 11.6.8  & Windows11 _ |
|**Browser Details**|_Chrome Version 117.0.5938.150 (Official Build) (64-bit) & Firefox 117.0_|
